packaging/installer/netdata-uninstaller.sh

```diff
@@ -296,30 +349,25 @@ quit_msg() {
   fi
 }
 
-user_input() {
-  TEXT="$1"
-  if [ "${INTERACTIVITY}" = "-i" ]; then
-    read -r -p "$TEXT" >&2
-  fi
-}
-
 rm_file() {
   FILE="$1"
   if [ -f "${FILE}" ]; then
-    run rm -v ${INTERACTIVITY} "${FILE}"
+    if user_input "Do you want to delete this file '$FILE' ? "; then
+	  run rm -v "${FILE}"
+	fi
```

Comment:
Indentation is off here, and the `rm` command is not guaranteed to work if running non-interactively.